src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react'
+import DownIcon from './dropdown.svg'
+import { DropdownContainer, DropdownHeader, DropdownItem, DropdownList } from './DropdownStyle'
+
+export interface DropdownProps{
+    title: string;
+    items: string[];
+    onOptionClicked: (item:string) => void;
+}
+export const  Dropdown = ({title, items}:DropdownProps) => {
+    const [isOpen, setIsOpen] = useState(false);
+    const [selectedOption, setSelectedOption] = useState(null);
+
+    const onOptionClicked = (value: any) => {
+        setSelectedOption(value)
+        setIsOpen(false)
+    }
+    const toggling = () => setIsOpen(!isOpen);
```

Comment:
When using the prev state, it's better to use a function as the parameter of the setState to avoid mistakes